guides/source/active_storage_overview.md

```diff
@@ -273,6 +273,25 @@ Call `avatar.attached?` to determine whether a particular user has an avatar:
 user.avatar.attached?
 ```
 
+In some cases you might want to override a default service for a specific attachment.
+It can be done in a following way:
+
+```ruby
+class User < ApplicationRecord
+  has_one_attached :avatar, service: :s3
+end
+```
+
+Since the provided value would be stored in the DB, there is a possible situation when
+configuration for the required service is missing. There is a special configuration option
+`Rails.application.configuration.active_storage.service_misconfiguration_behavior` to control
+this behavior: `:log` (default) puts the warning to the log, `:fallback_to_default` would try
+to use service from `Rails.application.config.active_storage.service`, `:raise` would cause a
+`RuntimeError`.
+
+NOTE: if you've initialized ActiveStorage before this feature was shipped, Rails would
+ask you to generate and execute one more migration (`rails active_storage:install && rails db:migrate`).
```

Comment:
We can remove this `NOTE` when follow this approach https://github.com/rails/rails/pull/34935/files#r248751180

Reply:
✔️